### PR TITLE
Weekly updates 4/22/2022

### PR DESCRIPTION
**Pull request by KreigD**
- Issuers JSON link updates from Lindsey
- FAQ copy additions/updates
- Issuers page copy and layout updates

### CODE REVIEW

File: en/faq.html [hunk 1]
```diff
@@ -317,7 +331,31 @@ <h1>
...
+                        <a name="Are-SMART-Health-Cards-only-for-Covid-19-vaccination-verification" class="anchor"></a>
+                        <p class="question">
+                            <a href="faq.html#Are-SMART-Health-Cards-only-for-Covid-19-vaccination-verification" class="question">Are SMART Health Cards only for COVID-19 vaccination verification? </a>
+                        </p>
+                        <p class="answer">
+                            No. The SMART Health Card Framework was designed to empower individuals with access to their healthcare data in secure, easy-to-access, equitable, and privacy-preserving ways. Developed prior to the COVID-19 pandemic by researchers at public and private institutions, the creators are honored that the technology has helped organizations and people worldwide at an urgent and critical time. The power of the SMART Health Cards Framework is its open-sourced standard that can be applied to many use cases beyond COVID-19. Research and planning is currently underway for uses like childhood immunization records. Visit this Harvard Medical School article for more information about the SMART Health Cards Framework.
```
Comment (jmandel) [inline en/faq.html:346]: This answer needs review. There are some factual and technical issues here. Do we want to edit inline in this pull request or out of band?

File: en/faq.html [hunk 2]
```diff
@@ -317,7 +331,31 @@ <h1>
...
+                        <a name="How-do-I-add-my-booster-doses" class="anchor"></a>
+                        <p class="question">
+                            <a href="faq.html#How-do-I-add-my-booster-doses" class="question">How do I add my booster dose(s)? </a>
+                        </p>
+                        <p class="answer">
+                            When you get an additional dose after receiving a SMART Health Card, it will not automatically reflect on your original QR code. You will need to delete your first one and then download another, which will add your additional dose(s) to your original vaccination record. It can take several weeks to get the information from your vaccination location into your state registry, so you might not see it yet if you were recently vaccinated. If you continue to have issues, you should contact the organization from whom you received your vaccination and SMART Health Card, or check with your state immunization registry. Here is a list of <a href="https://www.cdc.gov/vaccines/programs/iis/contacts-locate-records.html#state" target="_blank">state registries</a> and here is a <a href="issuers.html">list of issuers</a>.
```
Comment (jmandel) [inline en/faq.html:358]: This answer also needs editing for clarity. It seems to assume that all health cards will be accessed in an electronic form and talks about deleting things, but we need to be clear that the story applies to paper as well, for example.

File: en/faq.html [hunk 3]
```diff
@@ -426,17 +464,20 @@ <h1>
                         <p class="answer">
                             SMART Health Cards are a way to store your vaccination history and/or test results for your personal records and can help you easily share your status to an organization that asks for it.
                         </p>
+                        <p class="answer">
+                            SMART Health Cards empower individuals with secure, equitable, and privacy-preserving access to their clinical information. As former Utah Governor Mike Leavitt has said about the benefit of SMART Health Cards, “I own the right to have my own data. And to have it in a form that I can present it conveniently for the kinds of things I want to do.”
```
Comment (jmandel) [inline en/faq.html:468]: We should review whether it is necessary to include a quotation here

**Review by jmandel — CHANGES_REQUESTED** [review]

**Comment by KreigD** [comment]
Closed to create corrected pull request

**Pull request closed without merging**